tests/assets/cloud_init.yaml

```diff
@@ -1,30 +1,14 @@
-# Note: This cloud-init doesn't use the standard cloud-init config probably due to: https://github.com/rancher/os2/issues/7
-# As a workaround, use the internal extended syntax
-stages:
-   network:
-     - name: "Setup users"
-       ensure_entities:
-       - path: /etc/passwd
-         entity: |
-            kind: "user"
-            username: "vagrant"
-            password: "x"
-            homedir: "/run/tmp/vagrant"
-            shell: "/bin/bash"
-       - path: /etc/shadow
-         entity: |
-            kind: "shadow"
-            username: "vagrant"
-            password: ""
-       - path: /etc/shadow
-         entity: |
-            kind: "shadow"
-            username: "root"
-            password: "ros"
-       commands:
-       - mkdir -p /run/tmp/vagrant
-     - name: "Setup pubkey"
-       authorized_keys:
-        vagrant:
-        - https://raw.githubusercontent.com/hashicorp/vagrant/main/keys/vagrant.pub
-name: "Setup for the vagrant user"
\ No newline at end of file
+#cloud-config
+
+# Add additional users or set the password/ssh keys for root
+users:
+- name: "root"
+  passwd: "ros"
+  ssh_authorized_keys:
+  - https://raw.githubusercontent.com/hashicorp/vagrant/main/keys/vagrant.pub
+- name: "vagrant"
+  passwd: "vagrant"
+  shell: "/bin/bash"
```

Comment:
Note, shell and homedir _are_ required or the user results invalid for the system. See #13

Reply:
I've added it to the docs here: https://github.com/rancher-sandbox/os2/pull/14/commits/1cdc879254dbf9e481ee257b48568b9c026adbc6 for the time being